pkg/controller/global/template/template_init_data.go

```diff
@@ -0,0 +1,273 @@
+			return errors.Wrapf(err, "Failed to create virtualMachineTemplateVersion %s:%s", vmTemplateVersion.Namespace, vmTemplateVersion.Name)
+		}
+	}
+	return nil
+}
+
+var (
+	initBaseTemplates = `
+apiVersion: harvester.cattle.io/v1alpha1
+kind: VirtualMachineTemplate
+metadata:
+  name: iso-image-base-template
+  namespace: {{ .Namespace }}
+spec:
+  description: Template for booting virtual machine from an ISO image
+---
+apiVersion: harvester.cattle.io/v1alpha1
+kind: VirtualMachineTemplate
+metadata:
+  name: raw-image-base-template
+  namespace: {{ .Namespace }}
+spec:
+  description: Template for booting virtual machine from a qcow2/raw image
+---
+apiVersion: harvester.cattle.io/v1alpha1
+kind: VirtualMachineTemplate
+metadata:
+  name: windows-iso-image-base-template
+  namespace: {{ .Namespace }}
+spec:
+  description: Template for booting Windows virtual machine from an ISO image
+`
+
+	initBaseTemplateVersions = `
+apiVersion: harvester.cattle.io/v1alpha1
+kind: VirtualMachineTemplateVersion
+metadata:
+  name: iso-image-base-version
+  namespace: {{ .Namespace }}
+spec:
+  templateId: {{ .Namespace }}:iso-image-base-template
+  vm:
+    running: true
+    template:
+      spec:
+        domain:
+          cpu:
+            cores: 1
+          devices:
+            disks:
+            - disk:
+                bus: virtio
+              name: rootdisk
+              bootOrder: 1
+            - cdrom:
+                bus: sata
+                readonly: true
+              name: cdrom-disk
+              bootOrder: 2
+          resources:
+            requests:
+              memory: 2048M
+        volumes:
+        - dataVolume:
+            name: datavolume-rootdisk
+          name: rootdisk
+        - dataVolume:
+            name: datavolume-cdrom-disk
+          name: cdrom-disk
+    dataVolumeTemplates:
+    - metadata:
+        name: datavolume-cdrom-disk
+      spec:
+        pvc:
+          accessModes:
+          - ReadWriteOnce
+          resources:
+            requests:
+              storage: 10Gi
+        source:
+          http:
+            url: ""
+    - metadata:
+        name: datavolume-rootdisk
+      spec:
+        pvc:
+          accessModes:
+          - ReadWriteOnce
+          resources:
+            requests:
+              storage: 10Gi
+        source:
+          blank: {}
+---
+apiVersion: harvester.cattle.io/v1alpha1
+kind: VirtualMachineTemplateVersion
+metadata:
+  name: raw-image-base-version
+  namespace: {{ .Namespace }}
+spec:
+  templateId: {{ .Namespace }}:raw-image-base-template
+  vm:
+    running: true
+    template:
+      spec:
+        domain:
+          cpu:
+            cores: 1
+          devices:
+            disks:
+            - disk:
+                bus: virtio
+              name: rootdisk
+              bootOrder: 1
+          resources:
+            requests:
+              memory: 2048M
```

Comment:
can u please double-check with @gitlawr whether the pod network config is required for windows VM on default?